<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>version</class>
 <widget class="QDialog" name="version">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="version_main">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>main</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="version_compile">
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
     <property name="text">
      <string>compile</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="version_git">
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
     <property name="text">
      <string>log</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ok">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok</sender>
   <signal>released()</signal>
   <receiver>version</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
